core-data/selectors.js

```diff
@@ -145,3 +145,28 @@ export const getEntityRecords = createSelector(
 export function getThemeSupports( state ) {
 	return state.themeSupports;
 }
+
+/**
+ * Returns the embed preview for the given URL.
+ *
+ * @param {Object} state    Data state.
+ * @param {string} url      Embedded URL.
+ *
+ * @return {Mixed} Undefined if the preview has not been fetched, false if the URL cannot be embedded, array of embed preview data if the preview has been fetched.
```

Comment:
The correct form of the mixed type in JSDoc is `{*}`.

See: http://usejsdoc.org/tags-param.html#multiple-types-and-repeatable-parameters

Reply:
However, it might be better to use the "nullable" type here.

http://usejsdoc.org/tags-type.html

This may not work so well given that it returns so many varying types, though this may identify that the selector is doing too much work. For example, should we consider a separate selector for `isURLEmbeddable` ?